src/components/AddMovie/styles.js

```diff
@@ -1,5 +1,11 @@
 import styled from "styled-components";
+import { Field, Form, ErrorMessage } from "formik";
+import { TiArrowSortedDown } from "react-icons/ti";
+import { FaCalendarAlt } from "react-icons/fa";
```

Comment:
Consider separating the import groups with an empty line.

Reply:
resolved